Echo.json5

```diff
@@ -90,7 +90,6 @@
     { name: 'AREnableQuickLinksAnimation2', value: true },
     { name: 'AREnableRedesignedSettings', value: true },
     { name: 'ARShowOnboardingPriceRangeScreen', value: true },
-    { name: 'AREnableFailingTestEcho', value: true },
```

Comment:
no need to keep this around since it was added and removed today for testing purposes and never made it to production